torch/cuda/amp/grad_scaler.py

```diff
@@ -191,7 +207,8 @@ def unscale_(self, optimizer):
         if not self._enabled:
             return
 
-        assert self._scale is not None, self._scale_not_initialized_error("unscale")
+        assert self._scale is not None, self._not_initialized_error("unscale_", "_scale")
```

Comment:
Nit: looks like these asserts come in pairs only, so you might as well have a single function for both of them